corehq/apps/enterprise/static/enterprise/js/enterprise_dashboard.js

```diff
@@ -11,7 +11,7 @@ hqDefine("enterprise/js/enterprise_dashboard", [
     kissmetrics
 ) {
     $(function () {
-        kissmetrics.track.event("[Enterprise Dashboard] Visited page");
+        kissmetrics.track.event("[Enterprise Console] Visited page");
```

Comment:
@biyeun is this a safe change? I do think its correct to switch over as much as possible, but I want to make sure we don't have any scripts or whatever that depend on event names

Reply:
yeah, I would not change this. just the user-facing text as the ticket asks @RynhardPietersen

Reply:
Ah ok, so I think there was a slight misunderstanding of the ticket. We want to rename the top-level section to "Enterprise Console" to make it distinct from the actual "Enterprise Dashboard" page. Keep the Enterprise Dashboard page as is...we are not intending to do a refactor here at all.